content/docs/user-guide/troubleshooting.md

```diff
@@ -53,3 +53,17 @@ Unable to detect supported link types, as the
 [cache directory](/doc/command-reference/config#cache) doesn't exist. It is
 usually created automatically by DVC commands that need it, but you can create
 it manually (e.g. `mkdir .dvc/cache`) to enable this check.
+
+## Unable to acquire lock {#lock-issue}
+
+You may encounter an error message saying `Unable to acquire lock` if you have
+another DVC process running in the project. If that is not the case, it usually
+means that DVC was terminated abruptly and manually removing the lock file in
+`.dvc/tmp/lock` should resolve the issue. If the issue still persists then it
+may be the case that you are running DVC on some network filesystem like NFS,
+Lustre, etc. If that is the case, the solution is to enable `core.hardlink_lock`
+which can be done by running following command:
```

Comment:
```suggestion
`.dvc/tmp/lock` should resolve the issue.

If the issue still persists then it may be the case that you are running DVC on
some network filesystem like NFS, Lustre, etc. If so, the solution is to enable
`core.hardlink_lock` which can be done by running following command:
```

Reply:
How about separating the paragraphs ^ @shcheklein ?

Reply:
I don't have any opinion on this @jorgeorpinel. Up to you, guys :)